### ISSUE TITLE
Unable to verify email because of MultipleObjectsReturned

### ISSUE THREAD

**Issue by peterbe** [mdn/kuma#7440]
https://sentry.prod.mozaws.net/operations/mdn-prod/issues/9436021/?referrer=github_plugin

**Comment by peterbe** [comment]
In the traceback you'll see:
```
  File "allauth/account/managers.py", line 25, in get_primary
    return self.get(user=user, primary=True)
```
Somehow this user has 2 primary email addresses. 

This could be due to historical reasons. The error happens deep in `allauth` but we could perhaps keep an eye on it happening to more people.

**Comment by peterbe** [comment]
Under the circumstances, this is only happening because of a "legacy user". The same username was attached to three different emails and I'm not sure this is common. I've talked to Axel about it and I've also helped me clean it up using the Django Admin. 

I've also set one of those "Ignore - Until it happens again - 10 times" in Sentry. 

With that in mind, let's leave it open in a limbo state until it happens again. It's very possible that it's not a pattern but just a handful of "odd" legacy users.
